SYCL/ESIMD/api/functional/ctors/ctor_array.hpp

```diff
@@ -96,6 +96,9 @@ template <typename DataT, typename SizeT, typename TestCaseT> class run_test {
 
 public:
   bool operator()(sycl::queue &queue, const std::string &data_type) {
+    if (!should_run_test_with_current_datatype<DataT>(queue.get_device())) {
+      return true;
+    }
```

Comment:
```suggestion
    if (!should_run_test_with<DataT>(queue.get_device())) {
      return true;
    }
```
As a suggestion.  Or maybe even
```suggestion
    if (should_skip_test_with<DataT>(queue.get_device())) {
      return true;
    }
```

Reply:
the function was renamed [here](https://github.com/intel/llvm-test-suite/pull/839/commits/1c069b5f4a52017db43d26010feca6a451457ef9#diff-ef0766a5c874c909fd25e4eec71893c4614c91112485144e4efcdab4d856d0a0R71)